mergify_engine/actions/request_reviews.py

```diff
@@ -165,6 +174,21 @@ async def run(
             except types.InvalidTeam as e:
                 team_errors.add(e.details)
 
+        for team, weight in self.config["users_from_teams"].items():
+            try:
+                await team.has_read_permission(ctxt)
+            except types.InvalidTeam as e:
+                team_errors.add(e.details)
+            else:
+                self.config["users"].update(
+                    {
+                        user: weight
+                        for user in await ctxt.repository.installation.get_team_members(
+                            team.team
+                        )
+                    }
+                )
+
```

Comment:
This adds all users from a single team. I doubt anyone wants that.
It's likely people wants a subset of N people from a team.

See https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team for what GitHub supports already.

Reply:
`users_from_teams` without settings `random_count` doesn't make sense.